packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/PdfViewerTextLayer.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { FC, useEffect, useRef, useState } from 'react';
+import cx from 'classnames';
+import PDFJSLib, { PDFPageProxy, PDFPageViewport, TextContent, TextContentItem } from 'pdfjs-dist';
+import { EventBus } from 'pdfjs-dist/lib/web/ui_utils';
+import { TextLayerBuilder } from 'pdfjs-dist/lib/web/text_layer_builder';
+
+const { RenderingCancelledException } = PDFJSLib as any;
+
+interface Props {
```

Comment:
Are these just a subset of the PdfViewer props? If so, should this just extend those? Something like:

```js
type Props = Pick<PdfViewerProps, 'loadedPage' | 'scale'> & { setTextLayerInfo?: ... };
```

Reply:
Only `scale` prop is so. I updated the type.

Reply:
Did this change? I see same `Props` object as previously (and my comment above still stands).

Reply:
Updated to use the same property both in `PdfViewer` and `PdfViewerTextLayer`